### PR TITLE
docker configuration

### PR DESCRIPTION
**Pull request by ovv**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+RUN mkdir /website /website/data /website/data/site
+WORKDIR /website
+
+# Upgrade System:
+RUN apt-get update && apt-get -y upgrade && apt-get -y autoremove && apt-get install -qq build-essential libssl-dev libxrender-dev wget git wkhtmltopdf libpango1.0-0 libcairo2 libpq-dev && rm -rf /var/lib/apt/lists/*
+
+RUN pip install --upgrade pip && pip install greenlet eventlet
+
+ADD requirements.txt requirements.txt
```
Comment (michael-k) [inline docker/Dockerfile]: Please use `COPY` instead of `ADD`

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+FROM python:2.7.9
+
+ENV PYTHONUNBUFFERED 1
+ENV PIP_NO_CACHE_DIR 0
+
+RUN mkdir /website /website/data /website/data/site
+WORKDIR /website
+
+# Upgrade System:
+RUN apt-get update && apt-get -y upgrade && apt-get -y autoremove && apt-get install -qq build-essential libssl-dev libxrender-dev wget git wkhtmltopdf libpango1.0-0 libcairo2 libpq-dev && rm -rf /var/lib/apt/lists/*
```
Comment (michael-k) [inline docker/Dockerfile]: Running `apt-get upgrade` is not recommended in Dockerfiles, see subsection *apt-get* in https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

**Comment by matrixise** [comment]
we need to find the credentials for the @europython account on hub.docker.com ;-)

**Comment by aramboi** [comment]
Thanks for your work in this PR. Some of these changes were merged as part of adding support for Docker in the local dev env (see: https://github.com/EuroPython/epcon/pull/1087). 

Right now we deploy the website in a docker container, however we don't rely on building an image and uploading it to a registry. Instead we build the image on the host and replace the running image with the newly built one (this is similar to how a PAAS like Heroku deploys an app).

We might look into building and deploying just the docker image, but that is not on the work group roadmap right now afaik. So for now, I will close this PR as it is outdated.

**Pull request closed without merging**